packages/rum/src/domain/segmentCollection/startDeflateWorker.spec.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  display,
+  isIE,
+  MonitoringMessage,
+  noop,
+  resetInternalMonitoring,
+  startFakeInternalMonitoring,
+} from '@datadog/browser-core'
+import { MockWorker } from '../../../test/utils'
+import { createDeflateWorker } from './deflateWorker'
+import { startDeflateWorker, resetDeflateWorkerState } from './startDeflateWorker'
+
+describe('startDeflateWorker', () => {
+  let deflateWorker: MockWorker
+  let createDeflateWorkerSpy: jasmine.Spy<typeof createDeflateWorker>
+  let callbackSpy: jasmine.Spy
+
+  beforeEach(() => {
+    deflateWorker = new MockWorker()
+    callbackSpy = jasmine.createSpy()
```

Comment:
```suggestion
    callbackSpy = jasmine.createSpy('callbackSpy')
```
same as https://github.com/DataDog/browser-sdk/pull/1139#discussion_r733507038 then